frontend/src/components/task/task.js

```diff
@@ -852,101 +907,163 @@ class Task extends Component {
                           <Chip
                             label=' $ 20'
-                            className={ classes.chip }
+                            color={ this.state.currentPrice === 20 ? 'primary' : '' }
                             onClick={ () => this.pickTaskPrice(20) }
                           />
                           <Chip
                             label=' $ 50'
-                            className={ classes.chip }
+                            color={ this.state.currentPrice === 50 ? 'primary' : '' }
                             onClick={ () => this.pickTaskPrice(50) }
                           />
                           <Chip
                             label=' $ 100'
-                            className={ classes.chip }
+                            color={ this.state.currentPrice === 100 ? 'primary' : '' }
                             onClick={ () => this.pickTaskPrice(100) }
                           />
                           <Chip
                             label=' $ 150'
-                            className={ classes.chip }
+                            color={ this.state.currentPrice === 150 ? 'primary' : '' }
                             onClick={ () => this.pickTaskPrice(150) }
                           />
                           <Chip
                             label=' $ 300'
-                            className={ classes.chip }
+                            color={ this.state.currentPrice === 300 ? 'primary' : '' }
                             onClick={ () => this.pickTaskPrice(300) }
                           />
                         </div>
 
-                        <FormControl fullWidth>
+                        <FormControl fullWidth style={ { marginTop: 15, marginBottom: 15 } }>
                           <InputLabel htmlFor='interested-amount'>
                             <FormattedMessage id='task.bounties.interested.amount.value' defaultMessage='Price' />
                           </InputLabel>
-                          <FormattedMessage id='task.bounties.interested.input.amount' defaultMessage='Price insert a value for this task' >
-                            { (msg) => (
-                              <Input
-                                id='interested-amount'
-                                startAdornment={ <InputAdornment position='start'>$</InputAdornment> }
-                                placeholder={ msg }
-                                type='number'
-                                inputProps={ { 'min': 0 } }
-                                value={ this.state.currentPrice }
-                                onChange={ this.handleInputInterestedAmountChange }
-                              />
-                            ) }
-                          </FormattedMessage>
+                          <Input
+                            id='interested-amount'
+                            endAdornment={ <InputAdornment position='start'>$</InputAdornment> }
```

Comment:
For some reason, the `InputAdorment` component should add to the beginning of the input, but in my case is showing on the end, can you take a look on this?
<img width="573" alt="Screen Shot 2019-11-03 at 7 48 16 PM" src="https://user-images.githubusercontent.com/88840/68090474-67a46800-fe74-11e9-8667-1a25653f3751.png">

Reply:
It's happened because frontend/public path is in .gitignore.

I think the absolute path is better than bundling assets but I am not sure. I will do it your way.

Reply:
In your design, there is no $ symbol. 

I added it with endAdornment because if I put startAdornment the placeholder stands at the top even its not focused.

Please check it here;
https://material-ui.com/components/text-fields/#input-adornments